install/mac.rst

```diff
@@ -308,13 +374,54 @@ this:
 
 .. code-block:: console
 
-   Brians-Mac:~ brian$ mpf --version
-   MPF v0.50.82
+   My-Mac:~ $ mpf --version
+   MPF v0.51.3
 
 (Note that the actual version number of your MPF installation will be whatever
 version is the latest.)
 
-6. Download & run the "Demo Man" example game
+5.2 Install MPF & MC (Latest)
+-----------------------------
+
+The stable release of MPF is updated every few months, but if you always want the most up-to-date
```

Comment:
Not sure if dev is right for most users. Maybe we could add a warning?

Reply:
Done!